man/fi_efa.7.md

```diff
@@ -82,6 +82,14 @@ No support for counters for the DGRAM endpoint.
 
 No support for inject.
 
+# PROVIDER SPECIFIC ENDPOINT LEVEL OPTION
+
+*FI_OPT_EFA_RNR_RETRY*
+: Defines the number of RNR retry. The application can use it to reset RNR retry
+  counter via the call to fi_setopt. Note that the reset must be before endpoint
```

Comment:
I'd suggest "Note that this option must be set before the endpoint is enabled."

Reply:
Sure, fixed in new rev.